frontend/src/component/Sidebar/WorkspaceListItem.jsx

```diff
@@ -45,20 +45,14 @@ class WorkspaceListItem extends React.Component {
   getIcon = () => {
     const { props } = this
 
-    const isDropActive = props.canDrop && props.isOver
+    const isDropAllowed = props.userRoleIdInWorkspace >= ROLE.contentManager.id
+    const isDropAllowedOnWorkspaceRoot = props.draggedItem && (props.draggedItem.workspaceId !== props.workspaceId || props.draggedItem.parentId !== 0)
 
-    if (isDropActive) {
-      const isDropAllowed = props.userRoleIdInWorkspace >= ROLE.contentManager.id
-      const isDropAllowedOnWorkspaceRoot = props.draggedItem && (props.draggedItem.workspaceId !== props.workspaceId || props.draggedItem.parentId !== 0)
-
-      if (isDropAllowed && isDropAllowedOnWorkspaceRoot) return <i className='fa fa-arrow-circle-down' />
-      return <i className='fa fa-times-circle' />
-    }
-
-    return props.label.substring(0, 2).toUpperCase()
+    if (isDropAllowed && isDropAllowedOnWorkspaceRoot) return 'fa-arrow-circle-down'
+    return 'fa fa-times-circle'
```

Comment:
I think it shouldn't return the '.fa' class since the return above doesn't return it and since the function usage on line 74 already have the '.fa' class

Reply:
Updated.